src/core/crypto/ecdsa.hpp

```diff
@@ -147,7 +133,7 @@ class P256
          * @retval kErrorFailed       Failed to generate key.
          *
          */
-        Error Generate(void);
+        inline Error Generate(void) { return otPlatCryptoEcdsaGenerateKey(this); }
```

Comment:
nit: you can simply remove the `inline` since implementations in hpp are already inline
```suggestion
        Error Generate(void) { return otPlatCryptoEcdsaGenerateKey(this); }
```

Reply:
got it, thanks